runtime/src/main/java/org/corfudb/runtime/view/AbstractReplicationView.java

```diff
@@ -24,6 +24,7 @@
  * Created by mwei on 12/11/15.
  */
 @Slf4j
+@Deprecated
```

Comment:
Ah, forget my question, I see you committed already org.corfudb.runtime.view.replication.ChainReplicationProtocol